docs/docs/prompts/optimize-prompt.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 5
+sidebar_label: Optimize Prompts
+---
+
+import { APILink } from "@site/src/components/APILink";
+
+# Optimize Prompts (Experimental)
+
+MLflow allows you to plug your prompts into advanced prompt optimization techniques through MLflow's unified interface using the <APILink fn="mlflow.genai.optimize_prompt" /> API.
+This feature helps you improve your prompts automatically by leveraging evaluation metrics and labeled data.
+Currently, DSPy's MIPROv2 algorithm is supported by this API.
+
+## Overview
+
+In order to use <APILink fn="mlflow.genai.optimize_prompt" /> API, you need to prepare the following:
+
+1. An existing prompt registered in MLflow.
```

Comment:
Let's link to prompt registry documentation for instruction

Reply:
Sure, I skipped prompt registry instructions since this page is linked below the basic prompt registry sections. But having a link looks good to me.